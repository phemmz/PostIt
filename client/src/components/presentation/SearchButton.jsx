import React from 'react';
import { Link } from 'react-router';

/**
 * Home
 * @return {*} div
 */
const SearchButton = () => {
  return (
    <div className="center">
      <Link
        to="/search"
        className="searchbtn waves-effect waves-light btn center search-users"
      >Search For Users</Link>
    </div>
  );
};

export default SearchButton;
