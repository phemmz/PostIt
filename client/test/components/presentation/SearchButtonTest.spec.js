import React from 'react';
import { shallow } from 'enzyme';
import SearchButton from '../../../src/components/presentation/SearchButton';

describe('SearchButton Component', () => {
  it('should render a Link button for searching users', () => {
    const wrapper = shallow(<SearchButton />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div').exists()).toBe(true);
    expect(wrapper.find('Link').exists()).toBe(true);
    expect(wrapper.find('div').length).toBe(1);
    expect(wrapper.find('Link').length).toBe(1);
  });
});
