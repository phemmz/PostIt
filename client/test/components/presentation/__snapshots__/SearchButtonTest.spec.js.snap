// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton Component should render a Link button for searching users 1`] = `
<div
  className="center"
>
  <Link
    className="searchbtn waves-effect waves-light btn center search-users"
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/search"
  >
    Search For Users
  </Link>
</div>
`;
